src/Secrets.php

```diff
@@ -14,6 +20,59 @@ class Secrets
      * @return void
      */
     public static function injectIntoEnvironment(string $path, array $parameters)
+    {
+        $parameters = \file_exists('/opt/extensions/AWSParametersAndSecretsLambdaExtension')
+            ? self::resolveUsingLambdaExtension($path, $parameters)
+            : self::resolveUsingAwsSdk($path, $parameters);
+
+        $injected = [];
+
+        foreach ($parameters as $key => $value) {
+            $injected[] = isset($_ENV[$key]) ? "{$key} (overwritten)" : $key;
+            $_ENV[$key] = $value;
+        }
+
+        if (! empty($injected)) {
+            fwrite(STDERR, 'Injected runtime secrets: ' . implode(', ', $injected) . PHP_EOL);
+        }
```

Comment:
> @tillkruss whats your feeling on this?

I haven't tested this, but LGTM.

Reply:
Should we also mention what injected the secrets?
- "via Lambda extension"
- "via Systems Manager SDK"

Reply:
That would allow us easily in the future to add AWS Secrets Manager as well.

Reply:
I guess you know where the secrets came from, so telling you the origin seems overkill te me.

And indeed, had future secrets manager in mind ;-)

Reply:
Ah right, I think we should make using the extension opt-in and keep the default SDK approach. In that scenario its nice to know which method is being used, no?